app/components/Post.js

```diff
@@ -1,7 +1,61 @@
 import React from "react";
+import { fetchItem, fetchComments } from "../utils/api";
+import queryString from "query-string";
+import { Link } from "react-router-dom";
+import { convertTime } from "../utils/helpers";
 
 export class Post extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      comments: [],
+      loading: true,
+      title: "",
+    };
+    this.updateComments = this.updateComments.bind(this);
+  }
+
+  componentDidMount() {
+    const { id } = queryString.parse(location.search);
+    fetchItem(id).then(({ kids, title }) => {
```

Comment:
you could maybe add a `.catch()` to catch any errors that might come about from this request.  no big deal really, but just a thought.
```suggestion
    fetchItem(id).then(({ kids, title }) => {
      this.setState({
        title,
      });
      // fetch comments ...
      }).catch((error) => {
        // do something with error, set state, show an error state
      })
    }
```